Client/Frontend/Settings/SearchSettingsTableViewController.swift

```diff
@@ -155,6 +158,16 @@ class SearchSettingsTableViewController: ThemedTableViewController {
                 cell.accessibilityIdentifier = AccessibilityIdentifiers.Settings.Search.customEngineViewButton
                 cell.textLabel?.text = .SettingsAddCustomEngine
             }
+        case .privateSession:
+            cell.textLabel?.text = .Settings.Search.PrivateSessionSetting
+            cell.textLabel?.numberOfLines = 0
+            let toggle = ThemedSwitch()
+            toggle.applyTheme(theme: themeManager.currentTheme)
+            toggle.addTarget(self, action: #selector(didToggleDisableSearchSuggestionsInPrivateMode), for: .valueChanged)
+            toggle.isOn = model.shouldDisablePrivateModeSearchSuggestions
+            cell.editingAccessoryView = toggle
+            cell.selectionStyle = .none
+            cell.accessibilityIdentifier = AccessibilityIdentifiers.Settings.Search.disableSearchSuggestsInPrivateMode
```

Comment:
not ideal implementation, but kept consistent with legacy code for now. It seems going forward we would want to use `BoolSetting` but that would require a major refactor.